ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Storage/PerasCertDB/Impl.hs

```diff
@@ -260,6 +274,7 @@ data PerasVolatileCertState blk = PerasVolatileCertState
   , pvcsLastTicketNo :: !PerasCertTicketNo
   -- ^ The most recent 'PerasCertTicketNo' (or 'zeroPerasCertTicketNo'
   -- otherwise).
+  , pvcsLatestCertSeen :: !(Maybe (WithArrivalTime (ValidatedPerasCert blk)))
```

Comment:
Same question here, shouldn't this be a field of PerasCertDB, not just the volatile part?